.github/workflows/restore_snapshot_database.yml

```diff
@@ -1,25 +1,72 @@
-name: Restore Snapshot DB from production DB
```

Comment:
could we avoid touching the snapshot please? this is a different workflow for a test db we don't need to change